uberenv.py

```diff
@@ -614,7 +618,7 @@ def clone_repo(self):
             # do a pull to make sure we have the latest
             os.chdir(pjoin(self.dest_dir,"spack"))
             sexe("git stash", echo=True)
-            sexe("git pull", echo=True)
+            res = sexe("git pull", echo=True)
```

Comment:
Bonus bugfix!

Reply:
Sorry this is out the scope of the PR description...

Reply:
oh, no... I guess I'll have to revert that one :)